### ISSUE TITLE
/storage/emulated/0/PocketMine-MP/crashdumps/Fri_Jan_3-16.27.36-UTC_2020.log

### ISSUE THREAD

**Issue by FrancisMart321** [ScerIO/PocketMine-MP-Android#133]
This error does not allow me to give me op to open a server in my minecraft, if I put op and my username does not open the minecraft server in the friends section of my ps4, please help

**Comment by thundergod2720** [comment]
Yeah I'm encountering the same issue please fix this pocket mine devs

**Comment by Rayter26** [comment]
Are you sure you typed your MICROSOFT id? And not your psn?

**Comment by FrancisMart321** [comment]
> Are you sure you typed your MICROSOFT id? And not your psn?

Yes, and I even changed my microsoft name but it didn't work either, it seems to me that it is an application problem, not minecraft, since in the server tab, in the middle rectangle, when turning on the server nothing appears to me, (no line) A month or so ago the application had worked for me, not because now it doesn't.

**Comment by Rayter26** [comment]
Well that sucks :/

Le jeu. 9 janv. 2020 4:58 p.m., FrancisMart321 <notifications@github.com> a
écrit :

**Comment by ViewedArtist399** [comment]
Hi
